### PR TITLE
04_00_ARC04_6_1

### PR DESCRIPTION
**Pull request by rschlaikjer**

### CODE REVIEW

File: script/1_0_2current_en_script/Arcueid/Day 4/04_00_ARC04_6_1.txt [hunk 1]
```diff
@@ -7,46 +7,46 @@ I don't want to go through something like that ever again.
 If I stay with Arcueid, the same thing will definitely happen again. If I don't want that, I have no choice but to go back to the mansion.
 <Page3>
 "――――――"
-...There's a big flaw in my choice.
+...There's a big flaw in my choice.//Maybe revisit
```
Comment (rschlaikjer) [inline script/1_0_2current_en_script/Arcueid/Day 4/04_00_ARC04_6_1.txt:10]: Reads awkwardly to me but the JP doesn't give much to work with. 

> 自分の選択には大きな欠陥がある。

My read is that he knows he's making a bad decision?

File: script/1_0_2current_en_script/Arcueid/Day 4/04_00_ARC04_6_1.txt [hunk 2]
```diff
@@ -80,43 +80,43 @@ Even though the sun has just set, it is as quiet as if it were midnight.
...
+Suddenly, the whole area is turned a bright red.
 <Page29>
 I hear a rumbling sound.
 It's coming from somewhere very close.
-My whole body is convulsing in extreme pain like a spring.
+My entire body leaps in pain like a released spring.
 I'm thrashing around again and again.
 My consciousness, my body, my vision, the world,
-are all spinning like an emergency light.
+are all spinning like an emergency light.//
```
Comment (rschlaikjer) [inline script/1_0_2current_en_script/Arcueid/Day 4/04_00_ARC04_6_1.txt:106]: These things don't seem to have a proper name, at least that I could find. Is there a more succinct phrase for these?

File: script/1_0_2current_en_script/Arcueid/Day 4/04_00_ARC04_6_1.txt [hunk 3]
```diff
@@ -80,43 +80,43 @@ Even though the sun has just set, it is as quiet as if it were midnight.
...
 "?"
-Just now, there was something in front of me, a black <thing|lump of meat>.
+Right now, right in front of me, is something resembling a black lump of meat.
 <Page25>
 "――――――"
-My consciousness froze.
+My consciousness freezes.
 Why are those here, I wonder.
-It's obvious, or so I've come to realize.
+It's obvious, I realize.
```
Comment (rschlaikjer) [inline script/1_0_2current_en_script/Arcueid/Day 4/04_00_ARC04_6_1.txt:88]: This & above have a matched pattern in JP, shortened to emulate

> なぜここに、という疑問。
> とうぜんだ、という理解。

File: script/1_0_2current_en_script/Arcueid/Day 4/04_00_ARC04_6_1.txt [hunk 4]
```diff
@@ -7,44 +7,44 @@ I don't want to go through something like that ever again.
...
+Even as a joke, that's in poor taste.
+I can't laugh off what she just said.
 The girl is staring at me, not seeming to care that I have taken it so seriously.
 <Page10>
-"I'm not joking, I'm talking based on experience. A grandpa of someone I knew stopped breathing after three hours from looking like that. If I had called an ambulance, I might have been able to save him."
+"I'm not joking, I'm speaking based on experience. The grandpa of someone I know stopped breathing three hours after his face started looking like that. If I had called an ambulance, maybe he could have been saved."
 <Page11>
 The girl doesn't come close to me.
-She keeps her distance of about a meter,
+While keeping a distance of about a meter,
```
Comment (yk1512) [inline script/1_0_2current_en_script/Arcueid/Day 4/04_00_ARC04_6_1.txt:38]: I'm slightly inclined to keeping the original line but changing the comma to a full stop.

Comment (rschlaikjer) [reply script/1_0_2current_en_script/Arcueid/Day 4/04_00_ARC04_6_1.txt:38]: Yeah I initially thought this one was a typo til I looked back at the script / the next page played. Changing to a full stop works for me.

File: script/1_0_2current_en_script/Arcueid/Day 4/04_00_ARC04_6_1.txt [hunk 5]
```diff
@@ -55,20 +55,20 @@ With a sudden return to normal life, I start to breathe properly again.
...
+"No, just going for a walk isn't really a problem."
 I don't have any desire to get a dog, but it's kind of sad to completely deny it.
 <Page18>
 "A walk....
 Your face is still pale. I suggest that you go home quickly and get some rest."
 <Page19>
 The girl walks away.
 "Huh...?"
-The moment she left, for some reason, it felt like her condition was many times more worse than mine.
+As she left, I somehow got the feeling that her own physical condition was in far worse shape than mine.
```
Comment (yk1512) [inline script/1_0_2current_en_script/Arcueid/Day 4/04_00_ARC04_6_1.txt]: I think it should be either 'I somehow got the feeling that her own physical condition was far worse than mine' or 'I somehow got the feeling that she was in far worse shape than me.'

Comment (yk1512) [inline script/1_0_2current_en_script/Arcueid/Day 4/04_00_ARC04_6_1.txt]: I think it should be either 'I somehow got the feeling that her own physical condition was far worse than mine' or 'I somehow got the feeling that she was in far worse shape than me.'

File: script/1_0_2current_en_script/Arcueid/Day 4/04_00_ARC04_6_1.txt (hunk 1 above)
Comment (Avuxo) [reply script/1_0_2current_en_script/Arcueid/Day 4/04_00_ARC04_6_1.txt:10]: _little_ bit different, but reads more naturally.
...There's a problem with that choice though.

File: script/1_0_2current_en_script/Arcueid/Day 4/04_00_ARC04_6_1.txt (hunk 4 above)
Comment (Avuxo) [reply script/1_0_2current_en_script/Arcueid/Day 4/04_00_ARC04_6_1.txt:38]: I agree with changing to a full stop, but the original is awkward and maybe grammatically incorrect. "She keeps her distance" is already a full phrase and adding the "of" to the end makes it... weird. It looks like it should be either "She keeps her distance" or "She keeps a distance of about a meter" as-is. I propose:
> She keeps her distance, staying about a meter away.

File: script/1_0_2current_en_script/Arcueid/Day 4/04_00_ARC04_6_1.txt [hunk 6]
```diff
@@ -7,46 +7,46 @@ I don't want to go through something like that ever again.
...
 "Do you want me to call an ambulance?
 In that case, lend me your phone. I don't want to use mine for this."
 <Page13>
 "―――,# Ha."
 A breath of relief escaped my lips at the unexpected offer.
-As the girl had said, I had been experiencing a malfunction that I just hadn't noticed.
+Just as the girl says, it seems that I had sunk into a malaise without noticing.
 <Page14>
-With a sudden return to normal life, I start to breathe properly again.
+I snap back to to the real world, and my breathing returns to normal.
```
Comment (Avuxo) [inline script/1_0_2current_en_script/Arcueid/Day 4/04_00_ARC04_6_1.txt:47]: oh, there goes gravity

File: script/1_0_2current_en_script/Arcueid/Day 4/04_00_ARC04_6_1.txt [hunk 7]
```diff
@@ -55,20 +55,20 @@ With a sudden return to normal life, I start to breathe properly again.
 Just as she says her farewells, the girl asks me a strange question.
 <Page17>
 "...Well, I was born with it, I guess.
-Ever since I was a kid, I get dizzy whenever I exercise a little."
+Ever since I was a kid, if I exercise even a little I'll end up getting dizzy."
```
Comment (Avuxo) [inline script/1_0_2current_en_script/Arcueid/Day 4/04_00_ARC04_6_1.txt:58]: a little, I'll end up

File: script/1_0_2current_en_script/Arcueid/Day 4/04_00_ARC04_6_1.txt [hunk 8]
```diff
@@ -55,20 +55,20 @@ With a sudden return to normal life, I start to breathe properly again.
...
+"No, just going for a walk isn't really a problem."
+I don't have any desire to get a dog, but it's kind of sad to completely deny the possibility.
 <Page18>
 "A walk....
 Your face is still pale. I suggest that you go home quickly and get some rest."
 <Page19>
 The girl walks away.
 "Huh...?"
-The moment she left, for some reason, it felt like her condition was many times more worse than mine.
+As she left, I somehow got the feeling that her own physical condition was far more dire than mine.
```
Comment (Avuxo) [inline script/1_0_2current_en_script/Arcueid/Day 4/04_00_ARC04_6_1.txt:68]: drop "own" in "her own physical condition".
"than mine" -> "than my own."

File: script/1_0_2current_en_script/Arcueid/Day 4/04_00_ARC04_6_1.txt (hunk 6 above)
Comment (rschlaikjer) [reply script/1_0_2current_en_script/Arcueid/Day 4/04_00_ARC04_6_1.txt:47]: Yeah this (and jump around when Shiki gets springy) came unbidden to my thoughts while editing

File: script/1_0_2current_en_script/Arcueid/Day 4/04_00_ARC04_6_1.txt [hunk 9]
```diff
@@ -80,43 +80,43 @@ Even though the sun has just set, it is as quiet as if it were midnight.
 This is the place where I came to check on the noisy dog the night I came back to the mansion.
 <Page24>
 "?"
-Just now, there was something in front of me, a black <thing|lump of meat>.
+Right now, right in front of me, is something resembling a black lump of meat.
```
Comment (Avuxo) [inline script/1_0_2current_en_script/Arcueid/Day 4/04_00_ARC04_6_1.txt:83]: "there's something resembling"

File: script/1_0_2current_en_script/Arcueid/Day 4/04_00_ARC04_6_1.txt (hunk 2 above)
Comment (Avuxo) [reply script/1_0_2current_en_script/Arcueid/Day 4/04_00_ARC04_6_1.txt:106]: maybe "siren lights"? I'm ok with emergency lights, google doesn't turn up any real name for them other than "flashers" which sucks.

**Review by Avuxo — APPROVED** [review]
lgtm pending minor changes

File: script/1_0_2current_en_script/Arcueid/Day 4/04_00_ARC04_6_1.txt (hunk 1 above)
Comment (rschlaikjer) [reply script/1_0_2current_en_script/Arcueid/Day 4/04_00_ARC04_6_1.txt:10]: Yeah, the other thing that felt natural was 'flaw in my reasoning' but it felt like a stretch

**Pull request merged**